### DISCUSSION TITLE
Is there a "Upload" and "Download All Option" option?

### DISCUSSION THREAD

**Discussion by SShwinA** [localsend/localsend#1727] [Discussion (English, International)]
Tried to send many files to another person and found that there was no _Download all_ option when I tried to share it via link.  

Similarly, an upload form would be useful for receiving files. 
Trebleshot on android had an upload form and download all as zip option. Had to send files to older relatives and strangers, and that was very useful. Asking them to install an app on their pc or phone would be difficult.    
Using links for send & receive would be good. Is there such an setting in localsend, like the Share via link option?

**Comment by OsmanKAYI** [comment]
I am absolutely in the same situation and trying to find a way to download all of the files :)
